src/public-assets/js/dependent-dropdown.js

```diff
@@ -0,0 +1,21 @@
+var Dropdown = (function ($) {
+
+    return {
```

Comment:
@kenny-wealth I don't think there's a need to return an object with the init method, a simple IIFE should do.

Reply:
Fixed